installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc

```diff
@@ -12,6 +12,24 @@ toc::[]
 * You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
 * You have read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
 
+[id="planning-a-bare-metal-cluster-for-ocp-virt"]
+== Planning a bare metal cluster for {VirtProductName}
+
+If you will use {VirtProductName}, it is important to be aware of several requirements before you install your bare metal cluster.
+
+* You must have multiple worker nodes at the time of cluster installation. The high availability (HA) capability flag is permanently set when a cluster is first installed. If you do not have multiple worker nodes, you cannot use live migration.
++
+[NOTE]
+====
+You can install {VirtProductName} on a single-node cluster, but single-node OpenShift does not support high availability.
+====
+
+* Live migration requires shared storage. Storage for {VirtProductName} must support and use the ReadWriteMany (RWX) access mode.
+
+* Networking hardware must support Single Root I/O Virtualization (SR-IOV); you can verify this by referring to the list of xref:../../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[supported devices]. Using supported hardware enables you to xref:../../virt/virtual_machines/vm_networking/virt-configuring-sriov-device-for-vms.adoc#virt-configuring-sriov-device-for-vms[configure an SR-IOV device for virtual machines].
```

Comment:
I install OpenShift Virtualization all the time without SR-IOV capable NICs. This bullet point may be better with a qualification, e.g. If you want SR-IOV, check here for support and here for instructions (badly paraphrased)